Datepicker.js

```diff
@@ -449,10 +449,7 @@ class Datepicker {
         submitButton.style.gridColumnStart = 3;
         submitButton.style.gridColumnEnd = 8;
         submitButton.addEventListener('click', function (event) {
-            if (!this.singleDate && this.dates.length === 2) {
-                this.onSubmit();
-            }
-            if (this.singleDate && this.dates[0]) {
+            if (this.dates[0]) {
```

Comment:
We need to remove this if/else statement as well, since we do not need any conditions for onSubmit to fire off. We want the onSubmit() method to fire every time the submit button is clicked regardless of the dates.

Reply:
Are you sure i should delete this statement? After this operation onSubmit() fires even if no date is selected and return empty array.

Reply:
Yes, remember we are attempting to seperate concerns and make the API as semantic as possible. 

"Submit" means the user is clicking the `Submit` button even if there are no values, therefore we must treat this as a "onSubmit" event even if there are no dates chosen.

Reply:
I agree.